Loop/Views/SettingsView.swift

```diff
@@ -89,19 +90,22 @@ extension SettingsView {
         
     private var therapySettingsSection: some View {
         Section(header: SectionHeader(label: NSLocalizedString("Configuration", comment: "The title of the Configuration section in settings"))) {
-            return NavigationLink(destination: TherapySettingsView(
-                viewModel: TherapySettingsViewModel(mode: .settings,
-                                                    therapySettings: viewModel.therapySettings,
-                                                    supportedInsulinModelSettings: viewModel.supportedInsulinModelSettings,
-                                                    pumpSupportedIncrements: viewModel.pumpSupportedIncrements,
-                                                    syncPumpSchedule: viewModel.syncPumpSchedule,
-                                                    chartColors: .primary,
-                                                    didSave: viewModel.didSave))) {
-                LargeButton(action: { },
+            LargeButton(action: { self.showTherapySettings = true },
                             includeArrow: false,
                             imageView: AnyView(Image("Therapy Icon")),
                             label: NSLocalizedString("Therapy Settings", comment: "Title text for button to Therapy Settings"),
                             descriptiveText: NSLocalizedString("Diabetes Treatment", comment: "Descriptive text for Therapy Settings"))
+                .sheet(isPresented: $showTherapySettings) {
+                    TherapySettingsView(
```

Comment:
I'm not seeing a title now for therapy settings (though it shows up fine for the pump simulator and Dexcom screens)

Reply:
Really?

![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-08-28 at 12 54 06](https://user-images.githubusercontent.com/308993/91610158-92910880-e92d-11ea-848a-0ea34e0cec12.png)

Shows fine for me...what am I missing?

Reply:
Hmm, showing up fine now when I re-checked-out. Probably had an incorrect branch checked out (I'm betting LoopKit)

Reply:
Yes `TherapySettingsView` is in `LoopKit[UI]`, so that would explain it 👍